code/game/machinery/vending.dm

```diff
@@ -994,8 +994,29 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
 					/obj/item/clothing/mask/joy = 1,
 					/obj/item/clothing/head/cueball = 1,
 					/obj/item/clothing/under/scratch = 1,
-          /obj/item/clothing/under/sailor = 1)
-	contraband = list(/obj/item/clothing/suit/judgerobe = 1,/obj/item/clothing/head/powdered_wig = 1,/obj/item/weapon/gun/magic/wand = 2,/obj/item/clothing/glasses/sunglasses/garb = 2, /obj/item/clothing/glasses/sunglasses/blindfold = 1, /obj/item/clothing/mask/muzzle = 2)
+          			/obj/item/clothing/under/sailor = 1,
+          			/obj/item/clothing/shoes/megaboots=1,/obj/item/clothing/gloves/megagloves=1,/obj/item/clothing/head/helmet/megahelmet=1,/obj/item/clothing/under/mega=1,
+					/obj/item/clothing/shoes/protoboots=1,/obj/item/clothing/gloves/protogloves=1,/obj/item/clothing/head/helmet/protohelmet = 1,/obj/item/clothing/under/proto = 1,
+					/obj/item/clothing/shoes/megaxboots = 1,/obj/item/clothing/gloves/megaxgloves = 1,/obj/item/clothing/head/helmet/megaxhelmet = 1,/obj/item/clothing/under/megax = 1,
+					/obj/item/clothing/shoes/joeboots = 1,/obj/item/clothing/gloves/joegloves = 1,/obj/item/clothing/head/helmet/joehelmet = 1,/obj/item/clothing/under/joe = 1,
+					/obj/item/clothing/under/vault = 1,
+					/obj/item/clothing/under/roll = 1,
+					/obj/item/clothing/head/clownpiece = 1,/obj/item/clothing/under/clownpiece = 1,/obj/item/clothing/suit/clownpiece = 1)
+	contraband = list(/obj/item/clothing/suit/judgerobe = 1,/obj/item/clothing/head/powdered_wig = 1,/obj/item/weapon/gun/magic/wand = 2,/obj/item/clothing/glasses/sunglasses/garb = 2, /obj/item/clothing/glasses/sunglasses/blindfold = 1, /obj/item/clothing/mask/muzzle = 2,
+					/obj/item/clothing/gloves/anchor_arms = 1,
+					/obj/item/clothing/suit/kaminacape = 1,
+					/obj/item/clothing/under/soldieruniform = 1,/obj/item/clothing/head/stalhelm = 1,/obj/item/clothing/suit/soldiercoat = 1,
+					/obj/item/clothing/under/officeruniform = 1,
+					/obj/item/clothing/head/naziofficer = 1,
+					/obj/item/clothing/suit/officercoat = 1,
+					/obj/item/clothing/head/panzer = 1,
+					/obj/item/clothing/under/russobluecamooutfit = 1,/obj/item/clothing/head/russobluecamohat = 1,
+					/obj/item/clothing/suit/russofurcoat = 1,/obj/item/clothing/head/russofurhat = 1,
+					/obj/item/clothing/under/rottensuit = 1,/obj/item/clothing/shoes/rottenshoes = 1,
+					/obj/item/clothing/head/helmet/biker = 1,/obj/item/clothing/under/bikersuit = 1,/obj/item/clothing/gloves/bikergloves = 1,
+					/obj/item/clothing/under/jacketsuit = 1,
+					/obj/item/clothing/head/helmet/richard = 1,
+					/obj/item/clothing/under/vault13 = 1)
```

Comment:
Alright, since its probably better to post a more direct rebuttal as a direct comment rather than spread across three different posts.

Space Station 13 is a game packed to the brim with memes and references. Some are so ingrained in what is accepted and normal (Security always has Civil Protection's stun batons and the janitor always wears Roger Wilco's iconic jumpsuit, for example). But it isn't just tied to broad references to popular media, /TG/ for example,  had Tramadol in the for longest time because of a spam-bot that would 'advertise' cheap Tramadol on their forums (It was even referenced with its pill bottle stating 'Buy Tramadol now' as its description).

To shorten what I am saying: If your gonna start denying stuff because its meme/reference clothing, your gonna have to start changing _a lot_ of iconic SS13 items.